.github/workflows/pytest_coverage.yml

```
@@ -0,0 +1,60 @@
+          path: htmlcov/
+
+      - name: Generate coverage.xml
+        if: ${{ github.event_name != 'workflow_dispatch' }}
+        run: coverage xml
+
+      - name: Show coverage in PR
+        if: ${{ github.event_name != 'workflow_dispatch' }}
+        uses: orgoro/coverage@v3.2
+        with:
+          coverageFile: coverage.xml
+          token: ${{ secrets.GITHUB_TOKEN }}
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)